blackjack.py

```diff
@@ -202,23 +202,20 @@ def get_winner(pt, ph, ht, hh):
         return True
     if pt > 21 and ht > 21:
         print(
-            f'Player 1: {render_hand(ph)}= {pt} \nThe House: {render_hand(hh)}= {ht} \nboth players Bust!')
+            f'Player 1: {render_hand(ph)}= {pt} \nPlayers 1 Busts!')
         print("")
         if __name__ == "__main__":
             main()
 
 # get_hos is "Hit or Stay" t = total, ht = house total, h = hand, th = the_house
 
 
-def get_hit_or_stay(t, ht, d, h, th):
-    get_hos = input(f'Your Total is {t} and The House has {ht}, would you like to Hit? ("y" or "n"): ')
+def get_hit_or_stay(t, d, h):
+    get_hos = input(f'Your Total is {t}, would you like to Hit? ("y" or "n"): ')
     if get_hos == 'y':
         x = deal(d)
-        hd = deal(d)
         h.append(x)
-        th.append(hd)
         t += card_point(x)
-        ht += card_point(hd)
     if get_hos == 'n':
```

Comment:
This should probably be `elif` because the `else` below will fire in the case when `if get_hos == 'y'` is `True`.